site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -0,0 +1,137 @@
+
+* <<What is WebAuthn>>
+* <<Configuring WebAuthn>>
+** <<Tenant Configuration>>
+*** <<Relying party Id>>
+*** <<Workflow settings>>
+** <<Application Configuration>>
+
+== What is WebAuthn
+
+WebAuthn is a W3C https://www.w3.org/TR/webauthn-2/[specification] that defines an API to create and use public-key credentials in web applications. Check out FusionAuth's link:/blog/2022/09/13/what-is-webauthn-why-do-you-care[WebAuthn blog post] for more information on the basics of WebAuthn and its security benefits.
+
+== Configuring WebAuthn
+
+You can enable WebAuthn in your FusionAuth instance with a few configuration changes. This section explains these configuration options and provides detail to help you choose the best options for your instance.
+
+=== Tenant Configuration
+The bulk of WebAuthn configuration happens at the tenant level, including Relying Party settings and authenticator attachment and user verification requirements for WebAuthn workflows. Use the WebAuthn tab on the add/edit tenant page to configure these options.
+
+image::core-concepts/tenant-configuration-webauthn-general.png[Tenant Configuration - WebAuthn,width=1200, role=top-cropped,bottom-cropped]
+
+The settings in this first section are applied to all applications in the tenant.
+
+[.api]
+[field]#Enabled#::
+This toggle must be enabled to use WebAuthn on this tenant.
+
+[field]#Relying party Id#::
+The Relying Party Id controls the scope of WebAuthn passkeys (i.e. which sites a WebAuthn passkey can be used to authenticate). More on this link:#relying-party-id[below].
+
+[field]#Relying party name#::
+The Relying Party name is a human-readable name that may be displayed by the browser or operating system during a WebAuthn ceremony. The value should be something that your users will recognize such as the name of your company or service. If the field is left blank, FusionAuth will default the value to [field]#Issuer# from the General tab during WebAuthn ceremonies.
+
+[field]#Debug enabled#::
+Enable this toggle to create event logs with detailed information on WebAuthn failures. This can be useful for troubleshooting WebAuthn integration issues.
+
+==== Relying party Id
+The Relying Party Id controls which sites a given passkey can be used on. A passkey can only be used with the same Relying Party Id it was registered with, and there are some constraints on which values are valid for this field based on the browser's URL during the WebAuthn ceremony. Leaving this field blank, which causes the WebAuthn client API to use the browser request origin's effective domain, should work for most use cases, but overriding the value can provide additional flexibility. Next, we'll cover the constraints imposed on this value by the browser and then how and why you may want to override the value.
```

Comment:
"the browser's URL" feels awkward. How about this

```suggestion
The Relying Party Id controls which sites a given passkey can be used on. A passkey can only be used with the same Relying Party Id it was registered with. There are constraints on valid values based on the website where the  WebAuthn registration ceremony was performed. Leaving this field blank, which causes the WebAuthn client API to use the browser request origin's effective domain, should work for most use cases, but overriding the value can provide additional flexibility.

Next, we'll cover the constraints imposed on this value by the browser and then how and why you may want to override the value.
```

Reply:
I used

> There are constraints on valid values based on the website where the WebAuthn ceremonies are performed.

because the constraints apply during both ceremonies.